{
	"id": "oval:org.opensuse.security:def:201912308",
	"version": "1",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2019-12308",
		"affected": {
			"family": "unix",
			"platform": [
				"openSUSE Tumbleweed"
			]
		},
		"reference": [
			{
				"ref_id": "Mitre CVE-2019-12308",
				"ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-12308",
				"source": "CVE"
			},
			{
				"ref_id": "SUSE CVE-2019-12308",
				"ref_url": "https://www.suse.com/security/cve/CVE-2019-12308",
				"source": "SUSE CVE"
			},
			{
				"ref_id": "SUSE-SU-2019:2034-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2019-August/005769.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-SU-2019:2257-1",
				"ref_url": "https://www.suse.com/support/update/announcement/2019/suse-su-20192257-1.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-SU-2019:2335-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2019-September/005894.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-SU-2024:2817-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2024-August/019139.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "openSUSE-SU-2019:1839-1",
				"ref_url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/3LGJSPCN3VEG2UJPYCUB6TU75JTIV2TQ/#3LGJSPCN3VEG2UJPYCUB6TU75JTIV2TQ",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "openSUSE-SU-2019:1872-1",
				"ref_url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/5XTP44JEOSNXRVW4JDZXA5XGMBDZLWSW/#5XTP44JEOSNXRVW4JDZXA5XGMBDZLWSW",
				"source": "SUSE-SU"
			}
		],
		"description": "\n    An issue was discovered in Django 1.11 before 1.11.21, 2.1 before 2.1.9, and 2.2 before 2.2.2. The clickable Current URL value displayed by the AdminURLFieldWidget displays the provided value without validating it as a safe URL. Thus, an unvalidated value stored in the database, or a value provided as a URL query parameter payload, could result in an clickable JavaScript link.\n    ",
		"advisory": {
			"from": "security@suse.de",
			"severity": "Moderate",
			"cve": [
				{
					"text": "CVE-2019-12308 at SUSE",
					"href": "https://www.suse.com/security/cve/CVE-2019-12308/",
					"impact": "medium",
					"cvss3": "6.1/CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N"
				},
				{
					"text": "CVE-2019-12308 at NVD",
					"href": "https://nvd.nist.gov/vuln/detail/CVE-2019-12308",
					"impact": "medium",
					"cvss3": "6.1/CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N"
				}
			],
			"bugzilla": [
				{
					"text": "SUSE bug 1136468",
					"href": "https://bugzilla.suse.com/1136468"
				}
			],
			"issued": {
				"date": "2021-10-02"
			},
			"updated": {
				"date": "2024-08-08"
			},
			"affected_cpe_list": {
				"cpe": [
					"cpe:/o:opensuse:tumbleweed"
				]
			}
		}
	},
	"criteria": {
		"operator": "AND",
		"criterias": [
			{
				"operator": "OR",
				"criterions": [
					{
						"test_ref": "oval:org.opensuse.security:tst:2009855854",
						"comment": "python310-Django-4.2.11-2.1 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009875019",
						"comment": "python310-Django4-4.2.14-1.1 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009855855",
						"comment": "python311-Django-4.2.11-2.1 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009875020",
						"comment": "python311-Django4-4.2.14-1.1 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009855856",
						"comment": "python312-Django-4.2.11-2.1 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009875021",
						"comment": "python312-Django4-4.2.14-1.1 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009642348",
						"comment": "python36-Django-3.2.7-2.3 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009642349",
						"comment": "python38-Django-3.2.7-2.3 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009642350",
						"comment": "python39-Django-3.2.7-2.3 is installed"
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.opensuse.security:tst:2009634834",
				"comment": "openSUSE Tumbleweed is installed"
			}
		]
	}
}
